bin/apt-dependencies.sh

```diff
@@ -157,6 +158,9 @@ if [ "$1" != "nojs" ]; then
     apt-get install --no-install-recommends -y couch-libmozjs185-dev
   fi
   # newer releases have newer libmozjs
+  if [ "${VERSION_CODENAME}" == "noble" ]; then
+    apt-get install --no-install-recommends -y libmozjs-102-dev libmozjs-115-dev
```

Comment:
We are not 102 or 115 ready for Spidermonkey...

Reply:
understood, but that's what noble comes with. the work to be compatible with sm 102/115 will happen in couchdb itself, and we'll need a CI image to test it in

Reply:
only informational: PR for [v102](https://github.com/apache/couchdb/pull/4305) and [v115](https://github.com/apache/couchdb/pull/5146).

Reply:
so, you want only prepare this already, but don't hook it into Jenkins atm?

Reply:
If possible?